apps/bfd-model/bfd-model-rif-samples/dev/synthea/synthea.properities-08092021

```diff
@@ -0,0 +1,270 @@
+# Starting with a properties file because it requires no additional dependencies
+
+exporter.baseDirectory = ./output/
+exporter.use_uuid_filenames = false
+exporter.subfolders_by_id_substring = false
+# number of years of history to keep in exported records, anything older than this may be filtered out
+# set years_of_history = 0 to skip filtering altogether and keep the entire history
+exporter.years_of_history = 10
+# split records allows patients to have one record per provider organization
+exporter.split_records = false
+exporter.split_records.duplicate_data = false
+exporter.ccda.export = false
+exporter.fhir.export = true
```

Comment:
@jawalonoski / @hadleynet : Do we actually need/want this? And would you mind reviewing the other properties in here? You understand them much better than I.